cpp/bench/CMakeLists.txt

```diff
@@ -64,17 +66,18 @@ if(BUILD_CUML_PRIMS_BENCH)
     prims/matrix_vector_op.cu
     prims/permute.cu
     prims/reduce.cu
-    prims/rng.cu
-    ../src/common/logger.cpp)  # because prims is header only!
+    prims/rng.cu)
 
   if(NOT CUB_IS_PART_OF_CTK)
     add_dependencies(prims_benchmark cub)
   endif(NOT CUB_IS_PART_OF_CTK)
   add_dependencies(prims_benchmark spdlog)
 
-  target_link_libraries(prims_benchmark ${CUDA_cublas_LIBRARY} benchmarklib)
+  target_link_libraries(prims_benchmark ${CUDA_cublas_LIBRARY} benchmarklib ${CUML_CPP_TARGET})
```

Comment:
perhaps also do one line per entry here as well for consistency

Reply:
Fixed